releases/UWP/net-native2.0/README.md

```diff
@@ -4,8 +4,16 @@ You can see what was included in each .NET Native 2.0 ([Microsoft.NETCore.Univer
 
 When using Visual Studio these packages require Visual Studio 2017 Version 15.4 or later.
 
-### 6.0.1 (October 5, 2017)
+### UWP 6.0.2 (.NET Native 2.0.1) (November 20, 2017)
+
+#### Quality fixes
+- Fixes an issue when building UWP apps that target RS3 and referenced the Microsoft Store Services SDK and  Microsoft Advertising SDK or Store Engagement SDK [514498]
+- Fixes Crashes due to an error in the compiler's garbage collection reporting (affects ARM apps built with .NET Native) [507618]
+- Improves performance for creating new New UWP Projects. [512775]
+- Fixes issues that enable Windows Application Packaging (Centennial) projects publishing scenarios [521438]
```

Comment:
I feel like we should leave out the Windows Application Packaging mention since that hasn't publicly RTM'ed yet.

Reply:
Is there a preview or any other public release? If not, I will remove this.

Reply:
It's currently in VS previews, but it will RTM with the next VS release